roles/ceph-common/templates/ceph.conf.j2

```diff
@@ -114,6 +114,12 @@ filestore split multiple = {{ filestore_split_multiple }}
 osd op threads = {{ osd_op_threads }}
 filestore op threads = {{ filestore_op_threads }}
 filestore max sync interval = {{ filestore_max_sync_interval }}
+{% if filestore_xattr_use_omap %}
+filestore xattr use omap = {{ filestore_xattr_use_omap }}
+{% elif osd_mkfs_type == "ext4" %}
```

Comment:
Yeah, I see now.  :)  I personally find this a bit misleading (if something tells me to use true/false, I'll use the YAML boolean not a string), but you are right that this convention seems to be used elsewhere in defaults files.